disco_aws_automation/disco_acm.py

```diff
@@ -6,27 +6,45 @@
 import boto3
 import botocore
 
+CERT_SUMMARY_LIST_KEY = 'CertificateSummaryList'
+CERT_ARN_KEY = 'CertificateArn'
+DOMAIN_NAME_KEY = 'DomainName'
+
 
 class DiscoACM(object):
     """
     A class to manage the Amazon Certificate Service
 
     """
+    WILDCARD_PREFIX = "*."
 
     def __init__(self, connection=None):
         self._acm = connection
 
+    def _in_domain(self, domain, dns_name):
```

Comment:
Don't think we can use `in` though since _dns_name_ **a.b.c** would then mistakenly match domain ***.a**... unless I'm misunderstanding how matching should work in that case as well 😝